<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>565</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Bauhaus 93</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="font">
    <font>
     <family>Bauhaus 93</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
	background-color: #1a162a;
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="font">
       <font>
        <family>Bauhaus 93</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget{
		background-color: #f8f4ff;
}</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="music">
       <attribute name="title">
        <string>Music</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QProgressBar" name="volume">
          <property name="font">
           <font>
            <family>Bauhaus 93</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="musicList">
          <property name="font">
           <font>
            <family>Bauhaus 93</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QComboBox{
	background-color: #ace3cf;
	selection-background-color: #ace3cf;
}</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="upVolume">
          <property name="font">
           <font>
            <family>Bauhaus 93</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	background-color: #ace3cf;
}</string>
          </property>
          <property name="text">
           <string>Boost the sound by 10%</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="settings">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="font">
           <font>
            <family>Bauhaus 93</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="title">
           <string>Audio</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QRadioButton" name="activateAudioBoost">
             <property name="font">
              <font>
               <family>Bauhaus 93</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Activate audio boost</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="deactivateAudioBoost">
             <property name="font">
              <font>
               <family>Bauhaus 93</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Deactivate audio boost</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
